aws/resource_aws_ec2_transit_gateway_peering_attachment_accepter.go

```diff
@@ -0,0 +1,205 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceAwsEc2TransitGatewayPeeringAttachmentAccepter() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsEc2TransitGatewayPeeringAttachmentAccepterCreate,
+		Read:   resourceAwsEc2TransitGatewayPeeringAttachmentAccepterRead,
+		Update: resourceAwsEc2TransitGatewayPeeringAttachmentAccepterUpdate,
+		Delete: resourceAwsEc2TransitGatewayPeeringAttachmentAccepterDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"peer_account_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"peer_region": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"peer_transit_gateway_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"tags": tagsSchema(),
+			"transit_gateway_attachment_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"transit_gateway_default_route_table_association": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
```

Comment:
Can you please explain why this argument is in this resource?

```suggestion
```

Reply:
Accidental copy from the attachment resource - fixed!